index.html

```diff
@@ -1209,6 +1262,13 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents MAY re-fetch and update a manifest for an installed web
```

Comment:
"MAY **, at any time,**"

Just to be clear that there is no specific circumstance that should trigger the update. It's allowed to happen whenever the UA wants.